### PR TITLE
reduce-log-level-compress

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
The compress and clean commands for FASTQ are some of the most bloated logs at CG. This PR aims to reduce the spam in this log.

### Changed

- Reduced the log level of several compress related commands

### CODE REVIEW

File: cg/apps/housekeeper/hk.py [hunk 1]
```diff
@@ -273,7 +277,7 @@ def get_latest_bundle_version(self, bundle_name: str) -> Version | None:
         """Get the latest version of a Housekeeper bundle."""
         last_version: Version = self.last_version(bundle_name)
         if not last_version:
-            LOG.warning(f"No bundle found for {bundle_name} in Housekeeper")
+            LOG.debug(f"No bundle found for {bundle_name} in Housekeeper")
```
Comment (Vince-janv) [inline cg/apps/housekeeper/hk.py:280]: This one I'm unsure of... Do you think it should warn if no bundle exists?

Comment (islean) [reply cg/apps/housekeeper/hk.py:280]: I am confused about the whole method tbh. So we have a wrapper for `self.last_version` but what is the difference between the two apart from the added logging?

Probably out of scope but I do not think there is a need to have both methods present. If we are keeping both, I would say we should log a warning if that is the only difference between the methods.

**Review by islean — APPROVED** [review]
👍

File: cg/apps/housekeeper/hk.py (hunk 1 above)
Comment (Vince-janv) [reply cg/apps/housekeeper/hk.py:280]: I assume there is proper error handling with relevant logging down the line if the method expects anything else than `None`

**Comment by Vince-janv** [comment]
Deployed in stage and prod

**Pull request merged**